web/packages/teleport/src/lib/util.ts

```diff
@@ -18,6 +18,18 @@
 
 import { AuthType } from 'teleport/services/user';
 
+import type { FocusEvent } from 'react';
+
+/**
+ * Checks if a focus event occured outside target element
+ */
+export const focusOutsideTarget = (
```

Comment:
i think we want to start naming files related to the functions, similar to [this](https://github.com/gravitational/teleport/tree/2e43e37e504e3ad0c1d782868b7782253c32cf76/lib/utils)

so perahps we can pull this func out and name the file `eventTarget.ts`? (or whatever name you come up with)

Reply:
Yeah, I'd have gone with something even more generic like "events", just so that the next person who wants to add another util related to events has a place for it already. We probably don't want to end up on the other side of this spectrum where we'd have a separate file for each util.